RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2276,6 +2276,47 @@ boolean isUnclePAvailable()
 	return !page_text.contains_text("You don't know where a desert beach is");
 }
 
+boolean isDesertAvailable()
+{
+	//Is this workaround still needed or is mafia correctly recognizing desert is available in koe?
```

Comment:
Looking at my last KoE run log (which was on the 9th of December 2019) this was still required.
I don't see anything in the KoE thread on the mafia forum so we should probably tell them it needs fixed.